### PR TITLE
[e2e] Make meaningful service name in test

### PR DESCRIPTION
**Pull request by MrHohn**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
While looking at the test logs from service controller, I found it hard to parse meaningful information out as I see tons of services named `"service"` or `"service1"`. This PR gives test service meaningful name to make the debug experience less painful.

**Which issue(s) this PR fixes**:

Fixes #NONE 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE

```

### CODE REVIEW

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve
Thanks :-)

**Comment by MrHohn** [comment]
@BenTheElder Thanks for the review :)
